src/Symfony/Component/Debug/Tests/Exception/FlattenExceptionTest.php

```diff
@@ -34,88 +34,94 @@ class FlattenExceptionTest extends TestCase
 {
     public function testStatusCode()
     {
-        $flattened = FlattenException::create(new \RuntimeException(), 403);
+        $flattened = FlattenException::createFromThrowable(new \RuntimeException(), 403);
```

Comment:
unless required, these changes should also be reverted, this will help merges as usual

Reply:
Understandable, although I'd prefer to keep the test as it is. I'll change it.